docs/samples/Microsoft.ML.Samples/Dynamic/LightGBM/LightGBMMulticlassClassification.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic.LightGBM
+{
+    class LightGbmMulticlassClassification
+    {
+        public static void LightGbmMulticlassClassificationExample()
+        {
+            // Create a general context for ML.NET operations. It can be used for exception tracking and logging,
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create in-memory examples as C# native class.
+            var examples = DatasetUtils.GenerateRandomMulticlassClassificationExamples(1000);
+
+            // Convert native C# class to IDataView, a consumble format to ML.NET functions.
+            var dataView = mlContext.Data.ReadFromEnumerable(examples);
+
+            // Create a pipeline. 
+            //  - Convert the string labels into key types.
+            //  - Apply LightGbm multiclass trainer
+            var pipeline = mlContext.Transforms.Conversion.MapValueToKey("LabelIndex", "Label")
+                        .Append(mlContext.MulticlassClassification.Trainers.LightGbm(labelColumn: "LabelIndex"))
+                        .Append(mlContext.Transforms.Conversion.MapValueToKey("PredictedLabelIndex", "PredictedLabel"))
+                        .Append(mlContext.Transforms.CopyColumns("Scores", "Score"));
```

Comment:
>.Append(mlContext.Transforms.CopyColumns("Scores", "Score") [](start = 24, length = 59)

why this one is needed?

Reply:
I took the example from
[https://github.com/dotnet/machinelearning/blob/b863ac2458f486834a9da3ecdd5a69f51f99db73/docs/samples/Microsoft.ML.Samples/Static/LightGBMMulticlassWithInMemoryData.cs#L11](https://github.com/dotnet/machinelearning/blob/b863ac2458f486834a9da3ecdd5a69f51f99db73/docs/samples/Microsoft.ML.Samples/Static/LightGBMMulticlassWithInMemoryData.cs#L11)
I see there are two scores in dataview after transformation 1) Scores 2) Score. If there is no copy from `Score` to `Scores`, all the probabilities are zero. Any reason why?

---
In reply to: [255259910](https://github.com/dotnet/machinelearning/pull/2476#discussion_r255259910) [](ancestors = 255259910)

Reply:
That sound like a bug.

---
In reply to: [255679272](https://github.com/dotnet/machinelearning/pull/2476#discussion_r255679272) [](ancestors = 255679272,255259910)

Reply:
[https://github.com/dotnet/machinelearning/issues/2542](https://github.com/dotnet/machinelearning/issues/2542) to track it

---
In reply to: [256220736](https://github.com/dotnet/machinelearning/pull/2476#discussion_r256220736) [](ancestors = 256220736,255679272,255259910)